interfaces/builtin/unity7.go

```diff
@@ -145,6 +145,12 @@ dbus (receive,send)
     member=Changed
     peer=(label=unconfined),
 
+dbus (send)
+    bus=session
+    interface=com.canonical.UrlLauncher.XdgOpen
+    peer=(label=unconfined),
```

Comment:
This was a strawman. We're renaming it to com.canonical.URLLauncher, and must implement filtering of URLs on the server end. Actually, we should probably rename it to com.canonical.SafeLauncher instead, with an OpenURL method. WDYT, @mvo5?